Tests/Marketplace/marketplace_services.py

```diff
@@ -508,11 +543,15 @@ def add_pack_type_tags(self, yaml_content, yaml_type):
         Returns:
             Doesn't return
         """
+        logging.info("adding pack type tags")
         if yaml_type == 'Integration':
             if yaml_content.get('script', {}).get('feed', False) is True:
                 self._is_feed = True
             if yaml_content.get('script', {}).get('isfetchevents', False) is True:
                 self._is_siem = True
+
+            self.is_data_source_pack(yaml_content)
```

Comment:
This name isn't good.
The intent is to have a `bool` function so this part will be functional - i.e. the boolean flag `_is_data_source` should be updated by the result of the function.
The other flag `_single_integration` can be handled outside of the new function for a straight forward implementation.
i.e.
```
self._is_data_source = self.is_data_source_pack(yaml_content)
if not yaml_content.get('deprecated', False):
    self._single_integration = False